scrubadub/detectors/spacy.py

```diff
@@ -24,16 +24,12 @@
 
 class SpacyEntityDetector(Detector):
     """Use spaCy's named entity recognition to identify possible ``Filth``.
-
```

Comment:
Why were all the new lines removed? Does this work with the documentation or does it break it?

Reply:
strangely when I reverted the changes to split the expand person titles out, the new lines disappeared. will add them back in